bodhi/server/templates/update.html

```diff
@@ -884,7 +885,8 @@ <h5 class="m-t-2">
               % if build.ci_url:
                 <a href="${build.ci_url}">${self.util.ci_status2html(build.ci_status.description) | n}</a>
               % else:
-                ${self.util.ci_status2html(build.ci_status.description) | n}
+                ${self.util.ci_status2html(build.ci_status.description
+                  if build.ci_status else None) | n}
```

Comment:
Would you like me to change approach?

Btw, if you have time for the tests, go for it, I'm not sure when I'll find some